charts/auctioneer/values.yaml

```diff
@@ -7,11 +7,11 @@ global:
 
 images:
   auctioneer:
-    repo: ghcr.io/astriaorg/astria-auctioneer
+    repo: ghcr.io/astriaorg/auctioneer
```

Comment:
Note, also bump up auctioneer's cpu and memory usage.

Reply:
increase the auctioneer cpu limits to 2 and cpu requests to 1.

Reply:
discussed offline and the outcome is that we want values in charts to be valid for local deployment only.